iroh-bitswap/src/lib.rs

```diff
@@ -127,16 +137,15 @@ pub trait Store: Debug + Clone + Send + Sync + 'static {
 impl<S: Store> Bitswap<S> {
     pub async fn new(self_id: PeerId, store: S, config: Config) -> Self {
         let network = Network::new(self_id);
-        let server = Server::new(network.clone(), store.clone(), config.server).await;
-        let client = Client::new(
-            network.clone(),
-            store,
-            server.received_blocks_cb(),
-            config.client,
-        )
-        .await;
-
-        // EVIL: must eventually not be unbounded !!!
```

Comment:
just to confirm, these channels look bounded to me, so it's right to remove this comment, but nothing has changed in this patch in terms of channel bounding, right?

Reply:
correct, it's just a cleanup, I had forgotten to remove the comment when I changed the channel size